src/Api/Value/Topic.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace KafkaRest\Api\Value;
+
+use JMS\Serializer\Annotation as Serializer;
+
+final class Topic
```

Comment:
Shouldn't we ban constructor usage? Seems like it's intended to be only created through SerializerInterface